src/js/containers/Submission.jsx

```diff
@@ -91,7 +91,11 @@ class SubmissionContainer extends Component {
           period={params.filing}
           userName={user.profile.name}
           institution={params.institution} />
-        {code > 2 ? <div className="FloatingRefile"><RefileButton id={params.institution} filing={params.filing} code={code}/></div> : null}
+        {code > 2 ? <RefileButton
+          id={params.institution}
+          filing={params.filing}
+          code={code}
+          class='float' /> : null}
```

Comment:
In this case it's just a `prop` passed to `RefileButton` and used [here](https://github.com/cfpb/hmda-platform-ui/pull/390/files#diff-06b62e091b18afbc21500ce204115596R4).